tcpclient/connectionsManagement_test.go

```diff
@@ -13,41 +12,93 @@ import (
 // We really need to refactor this test. We should verify connections do become established,
 // rather than just waiting for a second and finish
 // We should also test "failing" connections, and ensure their status is reported properly
-func TestTcpConnect(t *testing.T) {
-	var numberConnections = 2
+func TestTcpConnectEstablished(t *testing.T) {
```

Comment:
To keep consistency... should we call this TestTCPConnectEstablished?

Reply:
yup!